File: treinamentos/Csharp-treinamento/WebAPIdesafio01/WebAPIdesafio01/Domain/Entity/Animal.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace WebAPIdesafio01.Domain.Entity
{
    public class Animal
    {
		public int IdAnimal { get; set; }

		public string Nome { get; set; }

		public int? Idade { get; set; }

		public string Especie { get; set; }

		public DateTime? DataNascimento { get; set; }

		public int? NivelFofura { get; set; }

		public int? NivelCarinho { get; set; }

		public string Email { get; set; }

	}
}

File: treinamentos/Java-treinamento/desafio01/src/main/java/br/com/iteris/universidade/desafio01/domain/entity/Animal.java
package br.com.iteris.universidade.desafio01.domain.entity;
import lombok.AllArgsConstructor;
import lombok.Data;

import java.time.LocalDate;
import java.util.Date;

@Data
@AllArgsConstructor
public class Animal {

    private Integer idAnimal;
    private String nome;
    private Integer idade;
    private String especie;
    private LocalDate dataNascimento;
    private Integer nivelFofura;
    private Integer nivelCarinho;
    private String email;

}
